Add configurable HAProxy health check parameters

timstoop opened the pull request:
**CHANGE DESCRIPTION**
---
**Problem:**

When a PXC node fails (e.g., during rolling restart), HAProxy takes 20+ seconds to detect the failure with default settings (`check inter 10000` + `fall 2` = 20s). Worse, existing client connections to the failed backend are NOT terminated, causing them to hang until TCP timeout (potentially minutes).

The only workaround is to provide a complete HAProxy configuration via `haproxy.configuration`, which duplicates operator logic, breaks on upgrades, and is difficult to maintain.

**Cause:**

HAProxy backend health check parameters (`interval`, `fall`, `rise`, `on-marked-down shutdown-sessions`) are hardcoded in the operator and not exposed through the CR API.

**Solution:**

Add a new `healthCheck` field to the HAProxy spec allowing granular control:
```yaml
haproxy:
  healthCheck:
    interval: 3000          # ms between checks (default: 10000)
    fall: 2                 # consecutive failures (default: 2)
    rise: 1                 # consecutive successes (default: 1)
    shutdownOnMarkDown: true # terminate connections (default: false)
```

This enables fast failover (6s vs 20s) and active connection cleanup without overriding the entire configuration.

**CHECKLIST**
---
**Jira**
- [ ] Is the Jira ticket created and referenced properly?
- [ ] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [ ] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [ ] Is an E2E test/test case added for the new feature/change?
- [x] Are unit tests added where appropriate?
- [ ] Are OpenShift compare files changed for E2E tests (`compare/*-oc.yml`)?

**Config/Logging/Testability**
- [x] Are all needed new/changed options added to default YAML files?
- [ ] Are all needed new/changed options added to the [Helm Chart](https://github.com/percona/percona-helm-charts)?
- [ ] Did we add proper logging messages for operator actions?
- [x] Did we ensure compatibility with the previous version or cluster upgrade process?
- [x] Does the change support oldest and newest supported PXC version?
- [x] Does the change support oldest and newest supported Kubernetes version?

**Additional Notes:**
- Fully backwards compatible (optional field with safe defaults)
- Comprehensive unit test coverage
- CRDs regenerated with validation rules

Fixes #2206

it-percona-cla commented:
[![CLA assistant check](https://cla-assistant.percona.com:443/pull/badge/not_signed)](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster-operator?pullRequest=2207) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/percona-xtradb-cluster-operator?pullRequest=2207) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/percona-xtradb-cluster-operator?pullRequest=2207) it.</sub>

JNKPercona commented:
| Test Name | Result | Time |
| ----------- | -------- | ------ |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/affinity-8-0.log) | 00:05:46 |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/auto-tuning-8-0.log) | 00:18:22 |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/cross-site-8-0.log) | 00:33:52 |
| custom-users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/custom-users-8-0.log) | 00:11:55 |
| demand-backup-cloud-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/demand-backup-cloud-8-0.log) | 00:15:52 |
| demand-backup-encrypted-with-tls-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/demand-backup-encrypted-with-tls-8-0.log) | 00:16:20 |
| demand-backup-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/demand-backup-8-0.log) | 00:16:20 |
| demand-backup-flow-control-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/demand-backup-flow-control-8-0.log) | 00:10:00 |
| demand-backup-parallel-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/demand-backup-parallel-8-0.log) | 00:09:04 |
| demand-backup-without-passwords-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/demand-backup-without-passwords-8-0.log) | 00:13:28 |
| haproxy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/haproxy-5-7.log) | 00:11:08 |
| haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/haproxy-8-0.log) | 00:11:05 |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/init-deploy-5-7.log) | 00:15:50 |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/init-deploy-8-0.log) | 00:16:27 |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/limits-8-0.log) | 00:13:23 |
| monitoring-2-0-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/monitoring-2-0-8-0.log) | 00:11:53 |
| monitoring-pmm3-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/monitoring-pmm3-8-0.log) | 00:10:45 |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/one-pod-5-7.log) | 00:14:37 |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/one-pod-8-0.log) | 00:14:11 |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/pitr-8-0.log) | 00:42:48 |
| pitr-gap-errors-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/pitr-gap-errors-8-0.log) | 00:54:23 |
| proxy-protocol-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/proxy-protocol-8-0.log) | 00:09:01 |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/proxysql-sidecar-res-limits-8-0.log) | 00:08:23 |
| pvc-resize-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/pvc-resize-5-7.log) | 00:14:39 |
| pvc-resize-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/pvc-resize-8-0.log) | 00:16:00 |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/recreate-8-0.log) | 00:16:42 |
| restore-to-encrypted-cluster-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/restore-to-encrypted-cluster-8-0.log) | 00:16:25 |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/scaling-proxysql-8-0.log) | 00:08:31 |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/scaling-8-0.log) | 00:10:46 |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/scheduled-backup-5-7.log) | 01:03:31 |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/scheduled-backup-8-0.log) | 01:02:18 |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/security-context-8-0.log) | 00:25:12 |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/smart-update1-8-0.log) | 00:32:03 |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/smart-update2-8-0.log) | 00:37:22 |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/storage-8-0.log) | 00:10:33 |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/tls-issue-cert-manager-ref-8-0.log) | 00:08:32 |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/tls-issue-cert-manager-8-0.log) | 00:11:14 |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/tls-issue-self-8-0.log) | 00:13:39 |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/upgrade-consistency-8-0.log) | 00:10:50 |
| upgrade-haproxy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/upgrade-haproxy-5-7.log) | 00:12:47 |
| upgrade-haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/upgrade-haproxy-8-0.log) | 00:12:29 |
| upgrade-proxysql-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/upgrade-proxysql-5-7.log) | 00:15:18 |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/upgrade-proxysql-8-0.log) | 00:15:47 |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/users-5-7.log) | 00:24:15 |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/users-8-0.log) | 00:25:57 |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/bc23e2f8/validation-hook-8-0.log) | 00:01:40 |
| We run 46 out of 46 | | 14:11:46 |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/2207/commits/bc23e2f8ab15486b07d7cb72a820a8a6566124c2
image: `perconalab/percona-xtradb-cluster-operator:PR-2207-bc23e2f8`

timstoop commented:
I totally agree that it makes sense to have it as default, it's what we were planning on doing in our Helm default code anyway. I only made it togglable in case there were good reasons that I didn't see and I really wanted to have this change implemented :-) I totally missed the 2205 PR, no idea how that could happen.
Do you want me to remove it from this patch and only use this PR to allow setting of the health checks instead?

JNKPercona commented:
| Test Name | Result | Time |
| ----------- | -------- | ------ |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/affinity-8-0.log) | 00:05:58 |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/auto-tuning-8-0.log) | 00:18:26 |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/cross-site-8-0.log) | 00:34:41 |
| custom-users-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/custom-users-8-0.log) | 00:11:53 |
| demand-backup-cloud-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/demand-backup-cloud-8-0.log) | 00:16:37 |
| demand-backup-encrypted-with-tls-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/demand-backup-encrypted-with-tls-8-0.log) | 00:16:47 |
| demand-backup-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/demand-backup-8-0.log) | 00:16:44 |
| demand-backup-flow-control-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/demand-backup-flow-control-8-0.log) | 00:10:12 |
| demand-backup-parallel-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/demand-backup-parallel-8-0.log) | 00:08:51 |
| demand-backup-without-passwords-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/demand-backup-without-passwords-8-0.log) | 00:32:39 |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/haproxy-5-7.log) | 00:13:34 |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/haproxy-8-0.log) | 00:13:59 |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/init-deploy-5-7.log) | 00:15:56 |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/init-deploy-8-0.log) | 00:16:45 |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/limits-8-0.log) | 00:11:50 |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/monitoring-2-0-8-0.log) | 00:22:27 |
| monitoring-pmm3-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/monitoring-pmm3-8-0.log) | 00:17:48 |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/one-pod-5-7.log) | 00:14:48 |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/one-pod-8-0.log) | 00:14:02 |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/pitr-8-0.log) | 00:45:43 |
| pitr-gap-errors-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/pitr-gap-errors-8-0.log) | 00:55:50 |
| proxy-protocol-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/proxy-protocol-8-0.log) | 00:09:03 |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/proxysql-sidecar-res-limits-8-0.log) | 00:08:25 |
| pvc-resize-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/pvc-resize-5-7.log) | 00:15:08 |
| pvc-resize-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/pvc-resize-8-0.log) | 00:16:57 |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/recreate-8-0.log) | 00:17:18 |
| restore-to-encrypted-cluster-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/restore-to-encrypted-cluster-8-0.log) | 00:16:48 |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/scaling-proxysql-8-0.log) | 00:08:14 |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/scaling-8-0.log) | 00:10:54 |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/scheduled-backup-5-7.log) | 01:04:11 |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/scheduled-backup-8-0.log) | 01:02:38 |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/security-context-8-0.log) | 00:27:07 |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/smart-update1-8-0.log) | 00:33:10 |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/smart-update2-8-0.log) | 00:37:47 |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/storage-8-0.log) | 00:10:33 |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/tls-issue-cert-manager-ref-8-0.log) | 00:08:38 |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/tls-issue-cert-manager-8-0.log) | 00:11:13 |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/tls-issue-self-8-0.log) | 00:13:15 |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/upgrade-consistency-8-0.log) | 00:10:52 |
| upgrade-haproxy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/upgrade-haproxy-5-7.log) | 00:12:32 |
| upgrade-haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/upgrade-haproxy-8-0.log) | 00:12:23 |
| upgrade-proxysql-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/upgrade-proxysql-5-7.log) | 00:14:54 |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/upgrade-proxysql-8-0.log) | 00:13:42 |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/users-5-7.log) | 00:25:41 |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/users-8-0.log) | 00:24:34 |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/41797862/validation-hook-8-0.log) | 00:01:38 |
| We run 46 out of 46 | | 15:03:29 |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/2207/commits/4179786287a222e27a3f3bbfba19ff24dd62507a
image: `perconalab/percona-xtradb-cluster-operator:PR-2207-41797862`

timstoop commented:
Removed on marked down shutdown sessions option, as requested.

Also fixed the handling of HA_SERVER_OPTIONS via a secret.

JNKPercona commented:
| Test Name | Result | Time |
| ----------- | -------- | ------ |
| affinity-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/affinity-8-0.log) | 00:07:14 |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/auto-tuning-8-0.log) | 00:21:04 |
| cross-site-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/cross-site-8-0.log) | 00:08:18 |
| custom-users-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/custom-users-8-0.log) | 00:07:25 |
| demand-backup-cloud-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/demand-backup-cloud-8-0.log) | 00:06:53 |
| demand-backup-encrypted-with-tls-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/demand-backup-encrypted-with-tls-8-0.log) | 00:12:01 |
| demand-backup-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/demand-backup-8-0.log) | 00:12:27 |
| demand-backup-flow-control-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/demand-backup-flow-control-8-0.log) | 00:35:43 |
| demand-backup-parallel-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/demand-backup-parallel-8-0.log) | 00:06:58 |
| demand-backup-without-passwords-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/demand-backup-without-passwords-8-0.log) | 00:39:53 |
| haproxy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/haproxy-5-7.log) | 00:47:29 |
| haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/haproxy-8-0.log) | 00:44:24 |
| init-deploy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/init-deploy-5-7.log) | 00:05:03 |
| init-deploy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/init-deploy-8-0.log) | 00:03:08 |
| limits-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/limits-8-0.log) | 00:09:54 |
| monitoring-2-0-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/monitoring-2-0-8-0.log) | 00:12:03 |
| monitoring-pmm3-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/monitoring-pmm3-8-0.log) | 00:13:29 |
| one-pod-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/one-pod-5-7.log) | 00:04:41 |
| one-pod-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/one-pod-8-0.log) | 00:04:37 |
| pitr-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/pitr-8-0.log) | 00:09:57 |
| pitr-gap-errors-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/pitr-gap-errors-8-0.log) | 00:10:02 |
| proxy-protocol-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/proxy-protocol-8-0.log) | 00:11:20 |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/proxysql-sidecar-res-limits-8-0.log) | 00:10:02 |
| pvc-resize-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/pvc-resize-5-7.log) | 00:44:22 |
| pvc-resize-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/pvc-resize-8-0.log) | 00:37:04 |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/recreate-8-0.log) | 00:17:51 |
| restore-to-encrypted-cluster-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/restore-to-encrypted-cluster-8-0.log) | 00:09:55 |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/scaling-proxysql-8-0.log) | 00:09:13 |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/scaling-8-0.log) | 00:13:29 |
| scheduled-backup-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/scheduled-backup-5-7.log) | 00:08:33 |
| scheduled-backup-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/scheduled-backup-8-0.log) | 00:08:08 |
| security-context-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/security-context-8-0.log) | 00:04:52 |
| smart-update1-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/smart-update1-8-0.log) | 00:09:31 |
| smart-update2-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/smart-update2-8-0.log) | 00:15:06 |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/storage-8-0.log) | 00:10:38 |
| tls-issue-cert-manager-ref-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/tls-issue-cert-manager-ref-8-0.log) | 00:05:00 |
| tls-issue-cert-manager-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/tls-issue-cert-manager-8-0.log) | 00:03:09 |
| tls-issue-self-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/tls-issue-self-8-0.log) | 00:45:01 |
| upgrade-consistency-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/upgrade-consistency-8-0.log) | 00:34:42 |
| upgrade-haproxy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/upgrade-haproxy-5-7.log) | 00:14:41 |
| upgrade-haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/upgrade-haproxy-8-0.log) | 00:14:47 |
| upgrade-proxysql-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/upgrade-proxysql-5-7.log) | 00:14:05 |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/upgrade-proxysql-8-0.log) | 00:25:08 |
| users-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/users-5-7.log) | 00:07:58 |
| users-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/users-8-0.log) | 00:12:05 |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/058ac6b2/validation-hook-8-0.log) | 00:02:05 |
| We run 46 out of 46 | | 11:51:49 |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/2207/commits/058ac6b2df9d7de7d63b0fda924169fee1893d8a
image: `perconalab/percona-xtradb-cluster-operator:PR-2207-058ac6b2`

timstoop commented:
Rebased and cleaned up the branch. Now contains a single commit with only the necessary changes:
- Added HAProxy health check configuration fields to the CR spec
- Generated CRDs updated by controller-gen v0.19.0 to reflect new fields
- Implementation in haproxy.go and haproxy_add_pxc_nodes.sh
- Test coverage for the new functionality

All generated files (CRDs, manifests) created with correct tool versions.

JNKPercona commented:
| Test Name | Result | Time |
| ----------- | -------- | ------ |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/affinity-8-0.log) | 00:06:08 |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/auto-tuning-8-0.log) | 00:19:31 |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/cross-site-8-0.log) | 00:35:54 |
| custom-users-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/custom-users-8-0.log) | 00:09:17 |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/demand-backup-cloud-8-0.log) | 00:57:35 |
| demand-backup-encrypted-with-tls-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/demand-backup-encrypted-with-tls-8-0.log) | 00:35:48 |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/demand-backup-8-0.log) | 00:43:08 |
| demand-backup-flow-control-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/demand-backup-flow-control-8-0.log) | 00:10:10 |
| demand-backup-parallel-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/demand-backup-parallel-8-0.log) | 00:09:22 |
| demand-backup-without-passwords-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/demand-backup-without-passwords-8-0.log) | 00:15:40 |
| haproxy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/haproxy-5-7.log) | 00:12:20 |
| haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/haproxy-8-0.log) | 00:12:01 |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/init-deploy-5-7.log) | 00:17:06 |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/init-deploy-8-0.log) | 00:17:38 |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/limits-8-0.log) | 00:12:14 |
| monitoring-2-0-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/monitoring-2-0-8-0.log) | 00:12:07 |
| monitoring-pmm3-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/monitoring-pmm3-8-0.log) | 00:11:40 |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/one-pod-5-7.log) | 00:14:36 |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/one-pod-8-0.log) | 00:15:11 |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/pitr-8-0.log) | 00:43:50 |
| pitr-gap-errors-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/pitr-gap-errors-8-0.log) | 00:57:24 |
| proxy-protocol-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/proxy-protocol-8-0.log) | 00:09:08 |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/proxysql-sidecar-res-limits-8-0.log) | 00:08:31 |
| pvc-resize-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/pvc-resize-5-7.log) | 00:17:28 |
| pvc-resize-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/pvc-resize-8-0.log) | 00:16:42 |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/recreate-8-0.log) | 00:16:55 |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/restore-to-encrypted-cluster-8-0.log) | 00:25:48 |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/scaling-proxysql-8-0.log) | 00:09:04 |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/scaling-8-0.log) | 00:11:23 |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/scheduled-backup-5-7.log) | 01:06:19 |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/scheduled-backup-8-0.log) | 01:02:16 |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/security-context-8-0.log) | 00:25:33 |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/smart-update1-8-0.log) | 00:33:02 |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/smart-update2-8-0.log) | 00:39:04 |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/storage-8-0.log) | 00:11:06 |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/tls-issue-cert-manager-ref-8-0.log) | 00:08:42 |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/tls-issue-cert-manager-8-0.log) | 00:11:48 |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/tls-issue-self-8-0.log) | 00:13:28 |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/upgrade-consistency-8-0.log) | 00:10:51 |
| upgrade-haproxy-5-7 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/upgrade-haproxy-5-7.log) | 00:11:43 |
| upgrade-haproxy-8-0 | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/upgrade-haproxy-8-0.log) | 00:13:06 |
| upgrade-proxysql-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/upgrade-proxysql-5-7.log) | 00:16:02 |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/upgrade-proxysql-8-0.log) | 00:15:12 |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/users-5-7.log) | 00:26:27 |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/users-8-0.log) | 00:26:02 |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/2ecd0074/validation-hook-8-0.log) | 00:01:50 |

| Summary | Value |
| ------- | ----- |
| Tests Run | 46/46 |
| Job Duration | 02:57:26 |
| Total Test Time | 16:16:31 |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/2207/commits/2ecd0074c7f1c435cc1dc6f0e6b897f46794aec9
image: `perconalab/percona-xtradb-cluster-operator:PR-2207-2ecd0074`

JNKPercona commented:
| Test Name | Result | Time |
| ----------- | -------- | ------ |
| affinity-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/affinity-8-0.log) | 00:00:00 |
| auto-tuning-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/auto-tuning-8-0.log) | 00:00:00 |
| backup-storage-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/backup-storage-tls-8-0.log) | 00:00:00 |
| cross-site-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/cross-site-8-0.log) | 00:00:00 |
| custom-users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/custom-users-8-0.log) | 00:00:00 |
| demand-backup-cloud-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/demand-backup-cloud-8-0.log) | 00:00:00 |
| demand-backup-encrypted-with-tls-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/demand-backup-encrypted-with-tls-8-0.log) | 00:40:18 |
| demand-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/demand-backup-8-0.log) | 00:00:00 |
| demand-backup-flow-control-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/demand-backup-flow-control-8-0.log) | 00:00:00 |
| demand-backup-parallel-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/demand-backup-parallel-8-0.log) | 00:00:00 |
| demand-backup-without-passwords-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/demand-backup-without-passwords-8-0.log) | 00:00:00 |
| haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/haproxy-5-7.log) | 00:00:00 |
| haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/haproxy-8-0.log) | 00:00:00 |
| init-deploy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/init-deploy-5-7.log) | 00:00:00 |
| init-deploy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/init-deploy-8-0.log) | 00:00:00 |
| limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/limits-8-0.log) | 00:00:00 |
| monitoring-2-0-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/monitoring-2-0-8-0.log) | 00:00:00 |
| monitoring-pmm3-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/monitoring-pmm3-8-0.log) | 00:00:00 |
| one-pod-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/one-pod-5-7.log) | 00:00:00 |
| one-pod-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/one-pod-8-0.log) | 00:00:00 |
| pitr-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/pitr-8-0.log) | 00:00:00 |
| pitr-gap-errors-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/pitr-gap-errors-8-0.log) | 00:00:00 |
| proxy-protocol-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/proxy-protocol-8-0.log) | 00:00:00 |
| proxysql-sidecar-res-limits-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/proxysql-sidecar-res-limits-8-0.log) | 00:00:00 |
| pvc-resize-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/pvc-resize-5-7.log) | 00:00:00 |
| pvc-resize-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/pvc-resize-8-0.log) | 00:00:00 |
| recreate-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/recreate-8-0.log) | 00:00:00 |
| restore-to-encrypted-cluster-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/restore-to-encrypted-cluster-8-0.log) | 00:00:00 |
| scaling-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/scaling-proxysql-8-0.log) | 00:00:00 |
| scaling-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/scaling-8-0.log) | 00:00:00 |
| scheduled-backup-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/scheduled-backup-5-7.log) | 00:00:00 |
| scheduled-backup-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/scheduled-backup-8-0.log) | 00:00:00 |
| security-context-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/security-context-8-0.log) | 00:00:00 |
| smart-update1-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/smart-update1-8-0.log) | 00:00:00 |
| smart-update2-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/smart-update2-8-0.log) | 00:00:00 |
| storage-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/storage-8-0.log) | 00:00:00 |
| tls-issue-cert-manager-ref-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/tls-issue-cert-manager-ref-8-0.log) | 00:00:00 |
| tls-issue-cert-manager-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/tls-issue-cert-manager-8-0.log) | 00:00:00 |
| tls-issue-self-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/tls-issue-self-8-0.log) | 00:00:00 |
| upgrade-consistency-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/upgrade-consistency-8-0.log) | 00:00:00 |
| upgrade-haproxy-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/upgrade-haproxy-5-7.log) | 00:00:00 |
| upgrade-haproxy-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/upgrade-haproxy-8-0.log) | 00:00:00 |
| upgrade-proxysql-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/upgrade-proxysql-5-7.log) | 00:13:26 |
| upgrade-proxysql-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/upgrade-proxysql-8-0.log) | 00:00:00 |
| users-5-7 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/users-5-7.log) | 00:00:00 |
| users-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/users-8-0.log) | 00:00:00 |
| validation-hook-8-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-2207/9d961162/validation-hook-8-0.log) | 00:00:00 |

| Summary | Value |
| ------- | ----- |
| Tests Run | 47/47 |
| Job Duration | 01:00:51 |
| Total Test Time | 00:53:45 |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/2207/commits/9d961162b3a6f058961a42b019e176250dd1c2d2
image: `perconalab/percona-xtradb-cluster-operator:PR-2207-9d961162`

egegunes commented on pkg/pxc/app/statefulset/haproxy_test.go:
```diff
@@ -108,3 +108,138 @@ func pointerToTrue() *bool {
+					Name: "test-cluster",
+				},
+				Spec: api.PerconaXtraDBClusterSpec{
+					CRVersion: "1.18.0",
```
please use `version.Version()` instead of hardcoding the version, this will fix unit test failures.

egegunes reviewed (dismissed):
LGTM but there are a few e2e test compare files that need to be updated.
